src/alert/alert.style.tsx

```diff
@@ -12,6 +12,10 @@ interface StyleProps {
     $type: AlertType;
     $sizeType: AlertSizeType;
 }
+export interface TextContentProps {
```

Comment:
We try to follow closely to the name of the styled component. So this should be `TextWrapperContainerStyleProps`